### ISSUE TITLE
Registration email to the volunteer doesn't include directions to WP

### ISSUE THREAD

**Issue by WaBirder57** [washingtontrails/vms#174]
When I registered for the Beacon Rock Trail WP (http://staging.wta.org:6543/workparty/a2eK0000000wRBp) the Thank you email didn't contain the directions to the work party - that section is blank.  The bottom part of the email is formatted as follows:

We have the following information:

Work party: Beacon Rock Trail
Date: 8/29/2015 - 8/29/2015
Days: 1
Name: Charlie KAhle

Directions to Work Party:

 

IF YOU WISH TO CANCEL please cancel the Work Party in your My Backpack or call the office.

**Comment by davisagli** [comment]
Is this a duplicate of #108?

**Comment by WaBirder57** [comment]
It may be related (probably) but if there isn't a fix to #108 then we should figure out how to do plain text if that is possible.

**Comment by davisagli** [comment]
I believe @drewpiston had a fix planned for #108 and just hasn't worked on it yet. Is that right Drew?

**Comment by drewpiston** [comment]
That's correct! Hoping to get to it tomorrow.

**Comment by davisagli** [comment]
I'm marking this as reviewing since Drew said #108 is done.

**Comment by WaBirder57** [comment]
This still doesn't seem to work.  I tested this by signing up to the Beacon Rock Trail workparty: http://staging.wta.org:6543/workparty/a2eK0000000wRBp/

The email I received didn't include the driving directions.  This is what the bottom part of the email looks like, note the Directions to Work Party is blank.

*****************************************
We have the following information:

Work party: Beacon Rock Trail
Date: 8/29/2015 - 8/29/2015
Days: 1
Name: Charlie Kahle


Directions to Work Party:

Breakfast Spot: Raging River Cafe
This is the regular Thursday work party, so will be meeting at 7:00 a.m. for breakfast at the Raging River Cafe, in Fall City. If you are coming from I-90, turn off at Preston exit 22, then drive to Fall City. At the intersection with the Redmond-Fall City Rd (Highway 202) turn left and go about 100 yards and the restaurant is on the left (south) side of the road. The address is 33723 Redmond-Fall City Rd. (Hwy 202), Fall City, Washington. All volunteers are welcome join us. It is a great opportunity to meet your co-workers. Otherwise you can meet us at the trailhead at the regular time.


IF YOU WISH TO CANCEL please cancel the Work Party in your My Backpack or call the office.


Washington Trails Association
www.wta.org

**Comment by drewpiston** [comment]
Hi @WaBirder57 ,

Sorry, I should have explained this more.   I think that's actually a data issue.

Basically, what I've done for this ticket is create code to sync a new field on the Work Party (called Trailhead Directions) to the field on the Trailhead (called Directions).  The code works fine for new records, but I neglected to do a data update for existing records.  Since that Work Party was created before this code was put in place, the new Trailhead Directions field was blank.

I went ahead and make an update to that Work Party so that now the Trailhead Directions field is populated, and the email should work.  I'll also update all the other records this morning.

 Drew
